docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -111,6 +111,14 @@ $ sudo sed -ri 's/.?group\s?=\s?".+"/group = "kvm"/1' /etc/libvirt/qemu.conf
 $ newgrp libvirt
 ----
 
+=== Start libvirtd service
+
+Start *libvirtd* service if not running.
+
+----
+$ sudo systemctl start libvirtd
```

Comment:
I'd recommend checking the status of **libvirtd** first via `systemctl is-active libvirtd`. If **libvirtd** is not active, start the service via `sudo systemctl start libvirtd`:

```adoc
. Check the status of **libvirtd**:
+
----
$ systemctl is-active libvirtd
----

. If **libvirtd** is not active, start the **libvirtd** service:
+
----
$ sudo systemctl start libvirtd
----
```

Reply:
Integrated your requested change, thanks.